go.mod

```diff
@@ -8,29 +8,30 @@ require (
 	github.com/go-stack/stack v1.8.1 // indirect
```

Comment:
Not sure what is doing that but we should stop casually updating a bunch of dependencies on every PR. When multiple people work on the same codebase it makes merging a nightmare and there are other unforeseeable things that might go unnoticed.

Reply:
I actually updated them in one place (I think?) and since I had to create a bunch of small PRs this one change started bouncing back and forth and causing trouble.